### DISCUSSION TITLE
KataGo AI for ConnectFour (Four in a row)

### DISCUSSION THREAD

**Discussion by hzyhhzy** [hzyhhzy/KataGo#19] [Announcements]
[https://en.wikipedia.org/wiki/Connect_Four](url)
This version supports all size of boards <= 20x20   
This version has much fewer wrong evaluations than the old version(20210511).   
   
The following chart is the results of every board size <= 20x20 calculated by KataGo    
Although these results are very sure, the possibility of errors cannot be completely ruled out.    
![image](https://github.com/user-attachments/assets/2913db11-2979-4dd8-b501-70f1758520d9)   

<hr /><em>This discussion was created from the release <a href='https://github.com/hzyhhzy/KataGo/releases/tag/ConnectFour_20241019'>KataGo AI for ConnectFour (Four in a row) </a>.</em>

**Comment by fawadkhanf3** [comment]
Hello. Can this be done for Connect 5 (5 in a row) for a 7x9 board (7 rows, 9 columns)?
